pref/qho.jl

```diff
@@ -0,0 +1,76 @@
+using QuBase
+using QuDynamics
+using BenchmarkLite
+
+type SampleProc{Alg} <: Proc end
+
+type Qexpmv end
```

Comment:
Mhm, after `using QuDynamics` you shouldn't need to do `type QuExpmV end`. This would redefine `QuExpmV` and you should get a warning. Otherwise it should work as you suggest. You use the method types defined in QuDynamics directly for `Alg` ...